src/ros_filter_bias_estimator.cpp

```diff
@@ -117,12 +118,23 @@ void RosFilterBiasEstimator::updateBiasEstimate(Eigen::Vector3d &orientation_mea
                         //  happens when the angle steps over the boundary (e.g. from -(PI-0.0001) to (PI-0.0001)). That is explained and handled in the
                         //  else condition.
                         double offset = FilterUtilities::clampRotation(uncorrected_orientation_estimate_[axis] - orientation_measurement[axis]);
+                        double difference_variance = 0.0;
+                        if(estimate_differences_.size() == num_values_variance_estimation_)
+                        {
+                            // Full data - can calculate
+                            double mean = 0.0;  // It is expecting a 0 offset between the two estimates if everything is working well.
```

Comment:
```suggestion
                            const double mean = 0.0;  // It is expecting a 0 offset between the two estimates if everything is working well.
```

mostly because at first I was looking for where the mean was calculated.

Reply:
If that's an issue, then I can just remove the variable. There is no point in labeling it const if the variable is created in the first place here.

Reply:
If it's removed can you please add a note that we use a 0 mean assumption there and that's why it is still variance instead of an average.

Reply:
It's removed